tests/testCameraMapper.py

```diff
@@ -34,10 +35,9 @@
 import lsst.afw.table as afwTable
 import lsst.daf.persistence as dafPersist
 import lsst.obs.base
-from lsst.utils import getPackageDir
 import shutil
 
-testDir = os.path.relpath(os.path.join(getPackageDir('obs_base'), 'tests'))
+testDir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
nit: We have called this ROOT in most other places. It would help to change that here so as not to confuse it with self.testDir, which is a temporary directory below this.